spec/deprecation_tracker_spec.rb

```diff
@@ -1,5 +1,9 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+require "date"
 require "tempfile"
-require_relative "spec_helper"
 require_relative "../lib/deprecation_tracker"
```

Comment:
This case we still need the `require_relatvie` because the deprecation_tracker class is out of the `next_rails` namespace. :+1: